src/NuGet.Core/NuGet.ProjectModel/PackageSpecExtensions.cs

```diff
@@ -8,6 +8,9 @@ namespace NuGet.ProjectModel
 {
     public static class PackageSpecExtensions
```

Comment:
>PackageSpecExtensions [](start = 24, length = 21)

It's not clear to me why these are extension methods rather than on the type itself? Unavailable parameter/returns types?